defaults/main.yml

```
@@ -6,21 +6,36 @@ certbot_auto_renew_hour: "3"
 certbot_auto_renew_minute: "30"
 certbot_auto_renew_options: "--quiet --no-self-upgrade"
 
+certbot_testmode: false
+certbot_hsts: false
+
 # Parameters used when creating new Certbot certs.
 certbot_create_if_missing: false
 certbot_create_method: standalone
 certbot_admin_email: email@example.com
+
+# Default webroot, overwritten by individual per-cert webroot directories
+certbot_webroot: /var/www/letsencrypt
+
 certbot_certs: []
-# - email: janedoe@example.com
-#   domains:
-#     - example1.com
-#     - example2.com
-# - domains:
-#     - example3.com
+#   - email: janedoe@example.com
+#     webroot: "/var/www/html/"
+#     domains:
+#       - example1.com
+#       - example2.com
+#   - domains:
+#       - example3.com
+
 certbot_create_command: >-
-  {{ certbot_script }} certonly --standalone --noninteractive --agree-tos
+  {{ certbot_script }} certonly --{{ certbot_create_method  }}
+  {{ '--hsts' if certbot_hsts else '' }}
+  {{ '--test-cert' if certbot_testmode else '' }}
+  --noninteractive --agree-tos
   --email {{ cert_item.email | default(certbot_admin_email) }}
+  {{ '--webroot-path ' if certbot_create_method == 'webroot'  else '' }}
+  {{ cert_item.webroot | default(certbot_webroot) if certbot_create_method == 'webroot'  else '' }}
```

```suggestion
  {{ cert_item.webroot | default(certbot_webroot) if certbot_create_method == 'webroot'  else '' }}
```

  {{ cert_item.webroot | default(certbot_webroot) if certbot_create_method == 'webroot' else '' }}